packages/synapse-react-client/src/components/widgets/query-filter/EnumFacetFilter/EnumFacetFilter.integration.test.tsx

```diff
@@ -26,25 +19,13 @@ import { createWrapper } from '../../../../testutils/TestingLibraryUtils'
 import QueryWrapper from '../../../QueryWrapper'
 import { mockTableEntity } from '../../../../mocks/entity/mockTableEntity'
 import { DEBOUNCE_DELAY_MS } from '../../../../utils/hooks/useImmutableTableQuery/useImmutableTableQuery'
-
-jest.mock('../../../../synapse-client', () => {
```

Comment:
Tests started failing due to this mock breaking. Improved the test surface by using the data provided by mock-service-worker instead of mocking SynapseClient